specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/locations.json

```diff
@@ -123,6 +162,140 @@
         }
       },
       "readOnly": true
+    },
+    "BillingResponseListResult": {
+      "description": "The response for the operation to get regional billingSpecs for a subscription.",
+      "properties": {
+        "vmSizes": {
+          "description": "The virtual machine sizes to include or exclude.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "vmSizeFilters": {
+          "description": "The virtual machine filtering mode. Effectively this can enabling or disabling the virtual machine sizes in a particular set.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VmSizeCompatibilityFilterV2"
+          }
+        },
+        "billingResources": {
+          "description": "The billing and managed disk billing resources for a region.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BillingResources"
+          }
+        }
+      }
+    },
+    "VmSizeCompatibilityFilterV2": {
+      "description": "This class represent a single filter object that defines a multidimensional set. The dimensions of this set are Regions, ClusterFlavors, NodeTypes and ClusterVersionsThe dimensions of this set are Regions, ClusterFlavors, NodeTypes and ClusterVersions. The constraint should be defined based on the following: FilterMode (Exclude vs Include), VMSizes (the vm sizes in affect of exlucsion/inclusion) and the ordering of the Filters. Later filters override previous settings if conflicted.",
```

Comment:
```suggestion
      "description": "This class represent a single filter object that defines a multidimensional set. The dimensions of this set are Regions, ClusterFlavors, NodeTypes and ClusterVersions. The dimensions of this set are Regions, ClusterFlavors, NodeTypes and ClusterVersions. The constraint should be defined based on the following: FilterMode (Exclude vs Include), VMSizes (the VM sizes in affect of exclusion/inclusion) and the ordering of the Filters. Later filters override previous settings if conflicted.",
```

Reply:
> 

Thanks for your suggestion. I have updated the description.